tests/test_policy.py

```diff
@@ -214,15 +214,15 @@ def test_securityhub_resource_support(self):
 
         whitelist = set(('AwsS3Object', 'Container'))
         todo = set((
-            # newer wave q3 2020
-            'AwsDynamoDbTable',
-            'AwsEc2Eip',
-            'AwsIamPolicy',
-            'AwsIamUser',
-            'AwsRdsDbCluster',
-            'AwsRdsDbClusterSnapshot',
-            'AwsRdsDbSnapshot',
-            'AwsSecretsManagerSecret',
+            # newer wave q3 2020 ~ these seem to use describe format
+            # 'AwsDynamoDbTable',
+            # 'AwsEc2Eip',
+            # 'AwsIamPolicy',
+            'AwsIamUser',  # this one has backwards compatible implications
+            # 'AwsRdsDbCluster',
+            # 'AwsRdsDbClusterSnapshot',
+            # 'AwsRdsDbSnapshot',
+            # 'AwsSecretsManagerSecret',
```

Comment:
I take it this is part of the "in progress" bit and will be cleaned up for landing.